annotator-core/src/main/java/edu/ucr/cs/riple/core/cache/downstream/DownstreamImpact.java

```diff
@@ -24,59 +24,51 @@
 
 package edu.ucr.cs.riple.core.cache.downstream;
 
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
 import edu.ucr.cs.riple.core.Report;
+import edu.ucr.cs.riple.core.cache.Impact;
 import edu.ucr.cs.riple.core.metadata.index.Error;
 import edu.ucr.cs.riple.core.metadata.index.Fix;
-import edu.ucr.cs.riple.core.metadata.method.MethodDeclarationTree;
 import edu.ucr.cs.riple.core.metadata.method.MethodNode;
+import edu.ucr.cs.riple.injector.location.OnMethod;
 import edu.ucr.cs.riple.injector.location.OnParameter;
+import java.util.Collection;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.stream.Collectors;
 
-/** Container class for storing overall effect of each method. */
-public class DownstreamImpact {
-  /** Node in {@link MethodDeclarationTree} corresponding to a public method. */
-  final MethodNode node;
+/**
+ * Container class for storing overall effect of fix in downstream dependencies. At this moment,
```

Comment:
Question: is this really about the impact of a fix **in** downstream dependencies (i.e. is the fix applied to the code of downstream dependencies) or is this the impact of a fix in the target **on** the downstream dependencies (errors that show up when building downstream dependencies due to a change in the code of the target)

Reply:
Yes this was not clear. It is the second case, the impact of a fix in the target on the downstream dependencies. Updated jvadoc clarity. thank you for noticing this. 21e7cefe122a7799ed6fa1b9c0d948f72ab7e215